src/book/Chapter4-Relations.lagda.md

```diff
@@ -96,7 +96,7 @@ _ = Set₉₈₆₀₂₅₀
 Why do these universes matter? As we have seen, Agda makes no distinction
 between values and types. So far, we've used this functionality primarily when
 working on indexed types, where we've seen values used as indices on types. But
-as we become more sophisticated Agda programmers (*eg.* later in this chapter,)
+as we become more sophisticated Agda programmers (*e.g.* later in this chapter),
```

Comment:
```suggestion
as we become more sophisticated Agda programmers (*e.g.* later in this chapter,)
```

Reply:
For whatever stupid reason, my style guide doesn't think English parses form a tree, thus this odd inside-out punctuation. But it is what it is.